.pipelines/e2e-step-template.yaml

```diff
@@ -90,6 +90,7 @@ steps:
       export REGIONS=$(AKS_ENGINE_REGION) 
       export IS_JENKINS=false 
       export DEBUG_CRASHING_PODS=true
+      export
```

Comment:
Why we need this one?

Reply:
To print out all the env variables. For better debuggability. This will mask any secrets like below:

```
declare -x CLIENT_ID="55990953-d723-433b-a204-01af59561ed8" 
declare -x CLIENT_SECRET="***" 
declare -x CLUSTER_DEFINITION="./cniLinux1604.json"
```